doc/api/console.md

```diff
@@ -162,6 +167,8 @@ console.log('this will also print');
 <!-- YAML
 added: v0.1.101
 -->
+* ```obj``` {any}
+* ```options``` {Object}
```

Comment:
FWIW, annotations for the available options, like what the documentations of [child_process. execFile()](https://github.com/nodejs/node/blob/master/doc/api/child_process.md#child_processexecfilefile-args-options-callback) does would be nice (Doesn't need to happen in this PR though)